docs/getting-started/ios/fastlane-swift.md

```diff
@@ -8,6 +8,9 @@ Fastlane.swift is currently in beta. Please provide feedback by opening an issue
 
 Fastlane.swift currently supports all built-in [fastlane actions](https://docs.fastlane.tools/actions/). Make sure to update to the most recent _fastlane_ release to try this feature.
 
+## PLugins 
+When you add any plugin fastlane the corresponding API will be automatically available in `fastlane/swift/Plugins.swift`
```

Comment:
```suggestion

When you add any plugin, _fastlane_ will generate the corresponding API and it will be available in `fastlane/swift/Plugins.swift`
```